tool/site/client/parsers/read/medication-request-parser.ts

```diff
@@ -2,7 +2,7 @@ import {MedicationRequest} from "../../models"
 import * as fhirExtension from "../../models/extension"
 
 export function getMedicationRequestNumberOfRepeatsAllowed(medicationRequest: MedicationRequest): string | number {
-  const repeatInformation = medicationRequest.extension.find(e =>
+  const repeatInformation = medicationRequest.dispenseRequest?.extension.find(e =>
```

Comment:
This extension is still in `MedicationRequest.extension`. It's just numberOfRepeatsAllowed which has moved to `MedicationRequest.dispenseRequest.numberOfRepeatsAllowed`. See `getRepeatsIssuedAndAllowed()` in `tool/site/client-react/src/fhir/helpers.ts`

Reply:
Moved